src/components/form/editForm/Form.edit.form.js

```diff
@@ -45,6 +45,22 @@ export default [
     key: 'revision',
     weight: 11,
   },
+  {
+    type: 'textfield',
+    input: true,
+    label: 'Form Revision (Deprecated)',
```

Comment:
Need to remove the edit form and just make it have an intelligent automatic fallback.